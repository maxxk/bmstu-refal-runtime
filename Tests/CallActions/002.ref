$ENTRY Go
{
	= <Prout <Func1 'A' 'F'>> <Prout<Func2 'A' 'F' 'S'>> <Prout<Func3 'A' 'F' 'S'>>
}

Func1
{
	e.1 s.1 'F' e.2, e.2 -> { s.3 = 1; 
								= 2;} : 2 = Sucess; 
}


Func2
{
	e.1 s.1 'F' e.2, e.2 -> { s.3 = 1; 
								= 2;} : 1 = Sucess; 
}

Func3
{
	e.1 s.1 'F' e.2, e.2 -> { s.3 = 1; 
								= 2;} : 3 = Sucess; 
}





