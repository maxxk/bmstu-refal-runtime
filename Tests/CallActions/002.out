Sucess 
Sucess 
Func call failed!
